src/docdb/tree/DocDBDatabaseTreeItemBase.ts

```diff
@@ -49,7 +50,7 @@ export abstract class DocDBDatabaseTreeItemBase extends DocDBTreeItemBase<Collec
     }
 
     public get connectionString(): string {
-        return `AccountEndpoint=${this.root.documentEndpoint};AccountKey=${this.root.masterKey};Database=${this.id}`;
+        return this.parent.connectionString.concat(`;Database=${this.id}`);
```

Comment:
`this.id` is the _tree_'s id and _happens_ to be the same as the database id, but that might change in the future. I would use `this._database.id` directly.